gdrive/export_client.py

```diff
@@ -172,3 +170,20 @@ def get_qualtrics_response(surveyId: str, responseId: str):
             f"No survey response found for responseId: {responseId}"
         )
     return r.json()
+
+
+def upload_spreadsheet(first, last, email, responseId, time):
```

Comment:
perhapps export upload_spreadsheet can be moved to client.py so the two client dont need to refrence each other. Also the function name seems wrong.

Reply:
Good idea, moved and name changed